src/ControlBarContainer/DashboardsBar.js

```diff
@@ -105,12 +105,9 @@ const DashboardsBar = ({
                 </div>
                 {dashboards.map(dashboard => (
                     <Chip
-                        key={dashboard.id}
```

Comment:
Is the key not needed anymore?

Reply:
No, since the dashboard chips are not <li> elements